src/isEmptyArray.js

```diff
@@ -0,0 +1,29 @@
+import { both, isEmpty } from 'ramda';
+
+import isArray from './isArray';
+
+
+/**
+ * Checks if input value is an empty `Array`.
+ *
+ * @func isEmptyArray
+ * @memberOf RA
+ * @since {@link https://char0n.github.io/ramda-adjunct/2.4.0|v2.4.0}
+ * @category Type
```

Comment:
Thinking if this still falls into `Type` category or is already an `Logic`. @guillaumearm what's your opinion on this ?

Reply:
It might be using logic under the hood, but its API and usage put it in Type imho.

Reply:
Agree

Reply:
In fact, it's a mix of both : Type and Logic

Because an empty array and a non empty array have the same type, it's a very good question to ask.
Is it ok to passing same type to a `Type` function produce a different result ?

on another side `RA.isNotEmpty` should be in `Logic` category like R.isEmpty

Reply:
Thanks for creating the issue for it. I’ll handle that. Regarding the question whether this is type or logic... I would say we are still in type world. If we look at it throug the lens of category theory let us say we have two new types **EmptyArray** and **NonEmptyArray**.  Assuming this we cas say that our two new type functions are just checking out the correct types. Logic of the checking becomes only the implementation detail.

Reply:
It seems correct, Agree ^^